src/styles/style.css

```diff
@@ -203,6 +203,17 @@ header nav ul {
   font-weight: bold;
 }
 
+@media (max-height: 450px) {
+  div.home {
+    height: 30px;
```

Comment:
It's to keep the scrollbar from showing when it's not wanted/needed on short height windows.

Reply:
:bulb: thanks!

Reply:
It might also work to delete `.home { height: calc(300px - var(--footer-height)); }` as well as this height?

(Apologies if that causes another problem I haven't spotted!)